apps/rush-lib/src/api/RushProjectConfiguration.ts

```diff
@@ -126,6 +126,9 @@ export class RushProjectConfiguration {
       }
     });
 
+  private static readonly _configCache: Map<RushConfigurationProject, RushProjectConfiguration | false> =
```

Comment:
The `RushProjectConfiguration | false` here is kinda confusing.

Reply:
Do you have a preferred method for explicitly storing that the cached config value is "does not exist"? I prefer to use primitives, but could use a Symbol or something instead.